core/src/main/java/lucee/runtime/functions/displayFormatting/GetLocaleInfo.java

```diff
@@ -66,6 +68,13 @@ private static Struct _call(PageContext pc, Locale locale, Locale dspLocale) {
 		iso.setEL(KeyConstants._country, locale.getISO3Country());
 		iso.setEL(KeyConstants._language, locale.getISO3Language());
 
+		Currency currency = Currency.getInstance(locale);
+		sct.setEL("currency_code", currency.getCurrencyCode());
+		sct.setEL("currency_symbol", currency.getSymbol(locale));
+
+		DateFormat dateFormat = DateFormat.getDateTimeInstance(DateFormat.FULL,DateFormat.SHORT,locale);
+		sct.setEL("Date & Time", dateFormat.format(new Date()));
```

Comment:
can you separate out the date and the time formats?

Reply:
Thanks for your suggestion. I've Separated the date & time formats.